### PR TITLE
net-misc/rclone-1.49.5 version bump

### PR DESCRIPTION
**Pull request by kuraga**
Signed-off-by: Alexander Kurakin <kuraga333@mail.ru>
Closes: https://bugs.gentoo.org/698036

### CODE REVIEW

File: net-misc/rclone/rclone-1.49.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+EGIT_COMMIT="v${PV}"
+SRC_URI="https://${EGO_PN}/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+inherit golang-vcs-snapshot
+
+DESCRIPTION="A program to sync files to and from various cloud storage providers"
+HOMEPAGE="https://rclone.org/"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
```
Comment (ConiKost) [inline net-misc/rclone/rclone-1.49.5.ebuild]: Drop it, if empty.

File: net-misc/rclone/rclone-1.49.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+DEPEND=""
```
Comment (ConiKost) [inline net-misc/rclone/rclone-1.49.5.ebuild]: Drop it, if empty.

File: net-misc/rclone/rclone-1.49.5.ebuild [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+RDEPEND="${DEPEND}"
```
Comment (ConiKost) [inline net-misc/rclone/rclone-1.49.5.ebuild]: Since DEPEND is empty, drop it.

File: net-misc/rclone/rclone-1.49.5.ebuild [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (ConiKost) [inline net-misc/rclone/rclone-1.49.5.ebuild]: Would be bump to EAPI=7?

Comment (kuraga) [reply net-misc/rclone/rclone-1.49.5.ebuild]: I'm not a specialist in EAPI 7 :)

**Comment by kuraga** [comment]
@ConiKost , done.

**Comment by kuraga** [comment]
Closed in favor of https://github.com/gentoo/gentoo/commit/1fc059708c69d941499c103f7ed78ac1f6e01cb5.

**Pull request closed without merging**